.github/pull_request_template.md

```diff
@@ -1,22 +1,28 @@
-# ✨ New in this PR ✨
+[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"
 
-## One liner: what did you do? 🚨 
-_give a brief overview of your PR_
+## What's new in this PR
+### Description
+[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
 
+### Screenshots
+[//]: # "Optional but strongly recommended - Add screenshots of expected behavior - GIFs if you're feeling fancy!"
```

Comment:
Add "specifically recommended for frontend PRs"?

Reply:
Good call -- will update.